responsibilities/community_meeting.md

```diff
@@ -0,0 +1,55 @@
+# Community Meeting
+
+At KubeVirt's present scale the community opts to host a weekly meetings for all interested contributors.  This gives contributors another channel to discuss important topics.  There is also value in adding a `Let's get to know each other` touch to project work.
+
+## Meeting Mechanics
+
+* Where: Zoom meeting ID: 92221936273
+  * Link to join is [here](https://zoom.us/j/92221936273)
+  * Community organizers will use CNCF/KubeVirt account to host meeting
+
+* When: Every Wednesday @ 16:00 CET (10:00 EST) calendar event
```

Comment:
time zones are sketchy. US just entered daylight savings so it's EDT for this half the year. I don't want to be overcomplicated about it, but should we think about ways to avoid confusion? maybe:

When: Every Wednesday @ 16:00 CET/CEST (10:00 EST/EDT) calendar event

Is this more confusing? :)